amd/src/embed.js

```diff
@@ -22,7 +22,10 @@ define(['jquery', 'mod_hvp/communicator'], function($, H5PEmbedCommunicator) {
                 var resizeDelay;
                 var instance = H5P.instances[0];
                 var parentIsFriendly = false;
-
+                var completionactivity = document.getElementsByClassName('activity-header')[0];
+                var comnpletionmargin = document.defaultView.getComputedStyle(completionactivity, '')
+                    .getPropertyValue('margin-bottom');
+                var completionheight = completionactivity.scrollHeight + parseInt(comnpletionmargin, 10); // We want the first element.
```

Comment:
Thanks @gbarat87 , this is now looking more robust. Just one more thing - what is 10? :)

Reply:
parseInt on base 10, but it's actually base 10 by default, so yeah, maybe I should remove that part

Reply:
Yeah, 10 means decimal. And as you found in https://developer.mozilla.org/en-US/docs/web/javascript/reference/global_objects/parseint#description _this does not always default to 10_ so I'm heapy to leave it as is. Thanks